export/exporter.go

```diff
@@ -591,3 +590,13 @@ func (e *exporter) emitItemOfReferences(outV string, inVs []string, docID string
 	id := e.nextID()
 	return id, e.emit(protocol.NewItemOfReferences(id, outV, inVs, docID))
 }
+
+func (e *exporter) emitMoniker(kind, sourceID, identifier string) error {
+	monikerID := e.nextID()
+	err := e.emit(protocol.NewMoniker(monikerID, kind, "go", identifier))
```

Comment:
```suggestion
	err := e.emit(protocol.NewMoniker(monikerID, kind, protocol.LanguageID, identifier))
```

~~Or if it's always "go", we could reduce one parameter and set this value inside `protocol.NewMoniker`.~~